doc/whats_new/v1.0.rst

```diff
@@ -140,6 +140,11 @@ Changelog
   descriptions in the generated HTML. :pr:`21493` by
   :user:`Aurélien Geron <ageron>`.
 
+- |Fix| Changes the error message of the ValidationError raised by
+  :func:`utils.validation.check_X_y` when y is None so that it is compatible
+  with :func:`utils.estimator_checks.check_requires_y_none`. :pr:`22578` by
+  :user:`Claudio Salvatore Arcidiacono <ClaudioSalvatoreArcidiacono>`.
```

Comment:
Can we move this into `doc/whats_new/v1.1.rst` ?

Reply:
Sure, it should have been moved now :)